index.html

```diff
@@ -47,10 +47,10 @@
     <!-- Live Date & Time -->
     <span id="span" class="d-flex align-items-center justify-content-center">Loading.....</span>
     <hr />
-    <img class="contest-img" src="images/leetcode.png" alt="Leetcode image">
     <div id="contests" class="d-flex flex-column container">
       <div class="card mb-2">
         <div class="card-body">
+          <img class="contest-img" src="images/leetcode.png" alt="Leetcode image">
```

Comment:
@deepesh16b Nice.
Now, can you programmatically change the icon based on the contest?

For now, whether it is any contest site, leetcode ICON will be displayed.

Reply:
Sorry I don't know how to do that, i am a newbie

Reply:
okay.